.github/workflows/generate.yml

```diff
@@ -24,6 +24,8 @@ jobs:
           cache-dependency-path: '**/*.sum'
       - name: Run 'go generate ./...'
         run: go generate ./...
+      - name: Run 'go mod tidy'
+        run: find . -iname go.mod | xargs dirname | xargs -I DIR go -C DIR mod tidy
```

Comment:
Does this work: `find -name go.mod -execdir go mod tidy \;`?
it's the same we have on dd-trace-go

Reply:
Duh I was not aware of `-execdir` (I haven't used this in a while, to be fair)